src/odemis/gui/conf/licences.py

```diff
@@ -0,0 +1,54 @@
+import configparser
+import logging
+import os
+from datetime import datetime
+
+from odemis.gui.conf.file import CONF_PATH
+
+
+# tmp flag for odemis advanced mode
+# TODO: remove and replace once the licenced version is released
+def get_license_enabled() -> bool:
+    """
+    Temporary function to get the license status.
+    Allows to enable/disable without changing the code.
+    """
+    enabled  = False
+    fibsem_enabled = False
+    milling_enabled = False
+    correlation_enabled = False
+    odemis_advanced_config = os.path.abspath(os.path.join(CONF_PATH, "odemis_advanced.config"))
+    if os.path.exists(odemis_advanced_config):
+        config = configparser.ConfigParser(interpolation=None)
+        config.read(odemis_advanced_config)
+        enabled = config["licence"].get("enabled", "False") == "True"
+        fibsem_enabled = config["licence"].get("fibsem", "True") == "True"
+        milling_enabled = config["licence"].get("milling", "True") == "True"
+        correlation_enabled = config["licence"].get("correlation", "True") == "True"
+
+        # expiry date
+        expires_at = config["licence"].get("expires_at", "") # YYYY-MM-DD
+        if expires_at:
```

Comment:
If it's not present, consider it's expired. To make it simple, you can put the default as "1970-01-01" (epoch).